plugins/inputs/sqlserver/azuresqlqueries.go

```diff
@@ -368,12 +446,25 @@ IF SERVERPROPERTY('EngineEdition') <> 5 BEGIN /*not Azure SQL DB*/
 	RETURN
 END
 
+DECLARE @tier AS nvarchar(128) = (SELECT TOP(1) edition from sys.database_service_objectives), 
+		@replica_id AS nvarchar(128) = (SELECT TOP(1) replica_id FROM sys.dm_database_replica_states), 
```

Comment:
Again here can we coalesce the 2 same queries for @replica_id and is_primary_replica to 1 query? Also use the is_local predicate?

Reply:
Updated.